src/components/modals/ProgressModal.vue

```diff
@@ -122,3 +124,22 @@ export default {
   },
 };
 </script>
+
+<style lang="scss" scoped>
+input[name='date'] {
+  display: none;
+}
+
+// Set specific width in order to match inlined datepicker.
+// TODO: Allow specyfing set sizes for modals in the future,
+// eg. wide, narrow, to keep these somewhat aligned.
+::v-deep .modal {
+  width: 341px;
```

Comment:
Hmm, det så ganske merkelig ut når modalen er så smal?

Burde det kanskje vært et sånt popup-felt i stedet for inline?

Reply:
Flatpickr som popup oppfører seg ikke helt 100% i modalen, og blir det ikke litt modal-i-modal? Lurer på om jeg foretrekker noe ala følgende selv, bl.a. færre trykk, men går vel an å få til..
<img width="789" alt="image" src="https://user-images.githubusercontent.com/2866225/197730786-17079e35-dbaf-4446-8d2f-4d50d216dc4e.png">

Reply:
Det så bedre ut enn den smale modalen IMO!